### PR TITLE
Rename package to qiskit-ibm-transpiler

### PR DESCRIPTION
**Pull request by jesus-talavera-ibm**
### Summary
Rename package from `qiskit-transpiler-service` to `qiskit-ibm-transpiler`

Fixes: #34 

Co-authored with: @y4izus

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
-# qiskit_transpiler_service
+# qiskit_ibm_transpiler
 
-A library to use [Qiskit Transpiler service](https://docs.quantum.ibm.com/transpile/qiskit-transpiler-service) and the [AI transpiler passes](https://docs.quantum.ibm.com/transpile/ai-transpiler-passes).
+A library to use [Qiskit Transpiler service](https://docs.quantum.ibm.com/transpile/qiskit-ibm-transpiler) and the [AI transpiler passes](https://docs.quantum.ibm.com/transpile/ai-transpiler-passes).
```
Comment (y4izus) [inline README.md:3]:
```suggestion
A library to use [Qiskit IBM Transpiler](https://docs.quantum.ibm.com/transpile/qiskit-ibm-transpiler) and the [AI transpiler passes](https://docs.quantum.ibm.com/transpile/ai-transpiler-passes).
```

File: README.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 
 **Note** The Qiskit transpiler service and the AI transpiler passes use different experimental services that are only available for IBM Quantum Premium Plan users. This library and the releated services are an alpha release, subject to change.
```
Comment (y4izus) [inline README.md:5]:
```suggestion
**Note** The Qiskit IBM Transpiler and the AI transpiler passes use different experimental services that are only available for IBM Quantum Premium Plan users. This library and the releated services are an alpha release, subject to change.
```

File: README.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 
-## Installing the qiskit-transpiler-service
+## Installing the qiskit-ibm-transpiler
 
-To use the Qiskit transpiler service, install the `qiskit-transpiler-service` package:
+To use the Qiskit transpiler service, install the `qiskit-ibm-transpiler` package:
```
Comment (y4izus) [inline README.md:9]:
```suggestion
To use the Qiskit IBM Transpiler, install the `qiskit-ibm-transpiler` package:
```

**Review by y4izus — CHANGES_REQUESTED** [review]
I realized we forgot to replace the "Qiskit Transpiler service" texts. I started creating suggestions, but I think it's easier if you do directly a replace :)

**Comment by jesus-talavera-ibm** [comment]
> I realized we forgot to replace the "Qiskit Transpiler service" texts. I started creating suggestions, but I think it's easier if you do directly a replace :)

Thank you @y4izus 
I just updated the PR

**Pull request merged**